docs/how-to/add-passkey.md

```diff
@@ -176,9 +176,10 @@ The **EMAIL** delivery method sends an email to the user. Clicking the link will
   />
 
 :::note
-You will need the user's `IDENTITY_ID` from identity creation above.  
-You can find the `REGION`, `TENANT_ID`, `REALM_ID` and `AUTHENTICATOR_CONFIG_ID` in your console.You can generate an `API_TOKEN` from you Beyond Identity Management API application.  
-Set the `POST_BINDING_REDIRECT_URI` to a URI in your application. On successful passkey binding the user will be re-directed to this URI.
+- You will need the user's `IDENTITY_ID` from identity creation above. This is the **id** returned in the response JSON from the create identity API.   
+- You can find the `REGION`, `TENANT_ID`, `REALM_ID` and `AUTHENTICATOR_CONFIG_ID` in your console.
+- You can generate an `API_TOKEN` from your [**Beyond Identity Management API application**](/docs/next/create-api-token#create-an-access-token-in-the-console) where the token contains the scope `credential-binding-jobs:create`.  
+- Set the `POST_BINDING_REDIRECT_URI` to a URI in your application. On successful passkey binding the user will be re-directed to this URI.
```

Comment:
For some reason, this last bullet isn't rendering. Also, could we make the curl request above into multiple lines like the Create Identity call?

<img width="1025" alt="Screenshot 2023-08-10 at 1 41 25 PM" src="https://github.com/gobeyondidentity/developer-docs/assets/15698998/426c44dd-5da2-4522-a7e5-23350b01c9dc">

Reply:
So there's actually tab that if you toggle it to the `EMAIL` option you should see this. When the tab is on `RETURN` then you won't see this https://developer-docs-git-sc-456-beyondidentity.vercel.app/docs/next/hosted-add-authentication?bind-delivery-method=email